src/tsconfig.json

```diff
@@ -0,0 +1,65 @@
+{
+    "compilerOptions": {
```

Comment:
Note: when we ship this, we can remove `jsconfig.json` (see #520) and merge its config into this file.